### PR TITLE
Daos container object movement using s3 cli

### PR DESCRIPTION
**Pull request by rajkumarpatel2602**
Added documentation for for daos container object movement from daos node to cortx and vice versa using aws s3 cli.
-----
[View rendered daos-cortx/docs/datamovment_with_s3.md](https://github.com/Seagate/cortx-experiments/blob/rajkumarpatel2602-patch-s3-cli-datamovement/daos-cortx/docs/datamovment_with_s3.md)

### CODE REVIEW

**Comment by johnbent** [comment]
@rajkumarpatel2602 , this is a good looking document.  Is it linked from another existing document yet?

**Review by johnbent — CHANGES_REQUESTED** [review]
Please link this new document from an existing document so that our pages remain nicely organized and good documents aren't lost/orphaned.

**Comment by rajkumarpatel2602** [comment]
> Please link this new document from an existing document so that our pages remain nicely organized and good documents aren't lost/orphaned.

Sure john. this was draft PR actually. So I would be fixing all the missing linkage. Thanks.

File: daos-cortx/docs/datamovment_with_s3.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+# Daos object movement using aws s3cli
+
+- Daos source container to Daos destination container object movement is possible using aws s3 cli. This documents is the setup guide to perform mentioned exercise.
```
Comment (kaustubh-d) [inline daos-cortx/docs/datamovment_with_s3.md:3]: Document describes data movement between DAOS and CORTX using Posix Interface on DAOS and S3 interface on CORTX. This demonstrates two workflows using AWS S3 cli tool.
1. DAOS to CORTX (posix to S3)
2. CORTX to DAOS (S3 to posix)

File: daos-cortx/docs/datamovment_with_s3.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+# Test setup
+
+## Prerequisites
+
+* Single node deployment of cortx on another VM
```
Comment (kaustubh-d) [inline daos-cortx/docs/datamovment_with_s3.md:9]: s/on another VM/on **one** VM/

File: daos-cortx/docs/datamovment_with_s3.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+    - The single node deployment for CORTX will hosting s3 server, which will be used to manage buckets and store objects requested aws s3 cli on daos node.
```
Comment (kaustubh-d) [inline daos-cortx/docs/datamovment_with_s3.md:11]: The single node deployment for CORTX hosting s3 server, which will be used to manage buckets and store objects moved from DOAS using aws s3 cli.

File: daos-cortx/docs/datamovment_with_s3.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+* Daos server node setup on a VM
+  
+    - Daos node will be required to create pools, containers and storing objects in it.
+
+* Install s3 cli on Daos node
+
+    - aws s3 cli Daos node will be used to storing/retrieving objects to/from CORTX node.
```
Comment (kaustubh-d) [inline daos-cortx/docs/datamovment_with_s3.md:19]: Aws s3 cli installed on DOAS node...

Comment (rajkumarpatel2602) [reply daos-cortx/docs/datamovment_with_s3.md]: done

File: daos-cortx/docs/datamovment_with_s3.md (hunk 2 above)
Comment (rajkumarpatel2602) [reply daos-cortx/docs/datamovment_with_s3.md]: done

File: daos-cortx/docs/datamovment_with_s3.md (hunk 1 above)
Comment (rajkumarpatel2602) [reply daos-cortx/docs/datamovment_with_s3.md]: done

File: daos-cortx/docs/datamovment_with_s3.md (hunk 3 above)
Comment (rajkumarpatel2602) [reply daos-cortx/docs/datamovment_with_s3.md]: done

File: daos-cortx/docs/datamovment_with_s3.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+        ls ~/.aws/credentials file`
+
+        config credentials
+
+- Copy contents from here and populate credentials and config files on daos node
+
+* Register domain name on daos node.
+
+- Added cortx node IP (s3.seagate.com) to /etc/hosts
```
Comment (kaustubh-d) [inline daos-cortx/docs/datamovment_with_s3.md:95]: use simple text instead of bullets

Comment (rajkumarpatel2602) [reply daos-cortx/docs/datamovment_with_s3.md]: done

File: daos-cortx/docs/datamovment_with_s3.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
+# Daos object movement using aws s3cli
+
+- Document describes data movement between daos and cortx using Posix Interface on daos and S3 interface on cortx. This demonstrates two workflows using AWS S3 cli tool.
```
Comment (kaustubh-d) [inline daos-cortx/docs/datamovment_with_s3.md:3]: remove bullet

Comment (rajkumarpatel2602) [reply daos-cortx/docs/datamovment_with_s3.md]: done

**Review by kaustubh-d — APPROVED** [review]

**Comment by rajkumarpatel2602** [comment]
> Please link this new document from an existing document so that our pages remain nicely organized and good documents aren't lost/orphaned.

John, I have added reference for this document inside main [README.md](https://github.com/Seagate/cortx-experiments/blob/main/daos-cortx/README.md) of daos-cortx integration project. This change will come in effect after merge.

**Comment by rajkumarpatel2602** [comment]
> @rajkumarpatel2602 , this is a good looking document. Is it linked from another existing document yet?

Yes.

**Review by kaustubh-d — APPROVED** [review]

**Pull request merged**